examples/triton-docker-combined/terraform.tfvars.example

```diff
@@ -0,0 +1,7 @@
+account = "bobby.brown"
```

Comment:
This is my ignorance, but what's the relationship between a `.tfvars` file and the `variables.tf` also in this PR?